src/tests/reservcounter.test.js

```diff
@@ -0,0 +1,27 @@
+import count from '../counterFunctions/reservationCounter.js';
+
+const reservations = [];
+
+const createObj = (id, username, dateStart, dateEnd) => ({
+  id,
+  username,
+  dateStart,
+  dateEnd,
+});
+
+const add = (obj) => {
+  reservations.push(obj);
+};
+
+test('Test Reservation counter function', () => {
+  add(createObj(1, 'kareem', '15-12-2022', '16-12-2022'));
+  expect(count(reservations)).toBe(1);
+});
+
+test('', () => {
+  add(createObj(1, 'kareem', '15-12-2022', '16-12-2022'));
+  add(createObj(2, 'kareem', '15-12-2022', '16-12-2022'));
+  add(createObj(3, 'kareem', '15-12-2022', '16-12-2022'));
+  add(createObj(4, 'kareem', '15-12-2022', '16-12-2022'));
+  expect(count(reservations)).toBe(5);
```

Comment:
- [ ] Good Job testing all of the functions of the counters 🎉 but please note that in `reservcounter.test.js`  you should test and cover all edge cases that can happen, also you tested the same function twice in two different tests block, which is unnecessary, instead I would suggest testing the case of having 0 items in the array as you guys did in the other counters testing, by implementing that you will cover all of the edge cases and you will avoid repeating same tests in this page  `reservcounter.test.js` , also it would be great if you guys group the tests inside a describe method 🙏 kindly fix that